pos_pricelist/static/src/js/models.js

```diff
@@ -513,6 +513,11 @@ function pos_pricelist_models(instance, module) {
                 }
                 break;
             }
+            //default price_extra is not included (product variants)
+            if(product['price_extra'] > 0.0){
+                price += product['price_extra']
+            }
+            
```

Comment:
i don't understand why you change this file. I guess it is out of the scope of this PR.
regards;